expression/builtin_compare_vec_test.go

```diff
@@ -29,6 +29,7 @@ var vecBuiltinCompareCases = map[string][]vecExprBenchCase{
 	ast.Coalesce: {},
 	ast.NullEQ: {
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETReal, types.ETReal}},
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString, types.ETString}},
```

Comment:
If each string is randomly generated, the probability that such two strings are equal will be very low. You can use ```randLenStrGener``` to generate a string of length 1 or 2 to increase the possibility of two strings being equal, to make the test more comprehensive.

Reply:
done.    I tried to use `randLenStrGener` to create two random strings with repetitive intervals

Reply:
Because the string is generated randomly. If the length of string is shorter, the more likely it is to generate the same string.